statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -273,10 +273,15 @@ func (wr *WidgetsRender) prepareListWithLogsForDisplay() {
 	wr.lLog.WrapText = true
 }
 
-func (wr *WidgetsRender) prepareLogLines(logData []string, size int) []string {
+func (wr *WidgetsRender) prepareLogLines(logData []string, maxSize int) []string {
```

Comment:
I would keep here the variable name "size" instead "maxSize", and I would modify line 278 in maxSize := size - 2

Reply:
done